client/src/components/mapping/TableViewer/index.tsx

```diff
@@ -110,10 +112,19 @@ const TableViewer = ({ table }: IProps) => {
           });
         });
     }
-  }, [resource, credential.owner, table, toaster]);
+  }, [resource, source.credential.owner, table, toaster]);
 
   return (
-    <React.Fragment>
+    <div id="tableViewer">
+      <StringSelect
+        icon={'th'}
+        inputItem={table}
+        items={tables}
+        maxItems={100}
```

Comment:
what if there are more than 100? if you start filtering, is it taken into account ?

Reply:
Yeah, I think so, we introduced that to avoid problems with DBs having too many tables